libs/vkd3d/device.c

```diff
@@ -3883,10 +3885,13 @@ static void STDMETHODCALLTYPE d3d12_device_GetCopyableFootprints(d3d12_device_if
         return;
     }
 
+    num_planes = max(vkd3d_popcount(format->vk_aspect_mask), 1);
     array_size = d3d12_resource_desc_get_layer_count(desc);
+    num_subresources_per_plane = desc->MipLevels * array_size;
+    num_subresources = num_subresources_per_plane * num_planes;
```

Comment:
We actually have `d3d12_resource_desc_get_sub_resource_count_per_plane` (and we arguably *should* have `d3d12_resource_desc_get_sub_resource_count` as well, but currently that one takes a resource rather than a resource desc).

Reply:
Fixed.